FreydCategoriesForCAP/gap/AdditiveClosure.gi

```diff
@@ -126,16 +126,50 @@ InstallMethod( AdditiveClosureMorphism,
     
 end );
 
+##
+InstallMethod( \/,
+               [ IsList, IsAdditiveClosureCategory ],
+               
+  function( list, category )
+    
+    if IsEmpty( list ) then
+      
+      Error( "The first argument should be a list of objects or listlist of morphisms!\n" );
+      
+    fi;
+    
+    if not IsCapCategoryObject( list[ 1 ] ) then
+      
+      TryNextMethod( );
```

Comment:
Is there a reason which I cannot see in this particular file why you use `TryNextMethod` instead of providing just a single method for `\/`? If no, I would be pleased if you could merge the two methods for `\/` into a single one. Doing so will lead to a more readable code for me.

Reply:
There is no particular reason for that. I have merged the two methods in one method.